src/firewall/functions.py

```diff
@@ -311,6 +311,17 @@ def checkIPnMask(ip):
                 return False
     return True
 
+def stripNonPrintableCharacters():
+    NOPRINT_TRANS_TABLE = {
+        i: None for i in range(0, sys.maxunicode + 1) if not chr(i).isprintable()
+    }
+    return NOPRINT_TRANS_TABLE
+
+def stripCharacters(rule_str):
+    table = stripNonPrintableCharacters()
+
+    return rule_str.translate(table)
```

Comment:
No need for this function after changing `stripNonPrintableCharacters()` as per other comments.